src/locales/en.json

```diff
@@ -1007,7 +1007,6 @@
     "gb": "GB",
     "gb-hours": "GB-hours",
     "gb-mo": "GB-month",
-    "hours": "hours",
```

Comment:
We need to leave this one here. It's not dead, but a value from an API

Reply:
done